volume/local/local.go

```diff
@@ -58,14 +60,18 @@ type Root struct {
 	volumes map[string]*Volume
 }
 
+// DataPath returns the constructed path of this volume.
 func (r *Root) DataPath(volumeName string) string {
 	return filepath.Join(r.path, volumeName, VolumeDataPathName)
 }
 
+// Name returns name of this (driver?)
```

Comment:
Name returns the name of Root, defined in the volume package in the `DefaultDriverName` constant.